api/src/main/java/me/lokka30/treasury/api/economy/account/Account.java

```diff
@@ -162,4 +205,51 @@ public void fail(@NotNull EconomyException exception) {
      */
     void retrieveHeldCurrencies(@NotNull EconomySubscriber<Collection<UUID>> subscription);
 
+    /**
+     * Request the {@link EconomyTransaction} history, limited by the {@code transactionCount}, of this {@code Account}.
+     *
+     * <p>If the specified {@code transactionCount} is higher than the known transactions, then this will return all the
+     * transactions.
+     * <p>If the specified {@code transactionCount} is {@code -1}, then this will return all the transactions. <i>If you
+     * really need all the transactions, please use this with care, because an economy provider might do a query to a database
+     * .</i>
+     *
+     * @param transactionCount the count of the transactions wanted
+     * @param subscription     the {@link EconomySubscriber} accepting the transaction history
+     * @author MrIvanPlays
+     * @since {@link me.lokka30.treasury.api.economy.misc.EconomyAPIVersion#v1_0 v1.0}
+     */
+    void retrieveTransactionHistory(
+            int transactionCount, @NotNull EconomySubscriber<Collection<EconomyTransaction>> subscription
```

Comment:
Maybe put each parameter in a diferent line?

Reply:
the editorconfig does these changes. that's really not necessary

Reply:
I thought it would split those line by default

Reply:
well the editorconfig is something which isn't ideal. it sometimes does and sometimes it tells such splits are "custom line splits". doesn't really matter tho as it is still readable like this.